private-templates-service/client/src/store/auth/actions.ts

```diff
@@ -0,0 +1,16 @@
+import { LOGIN, LOGOUT, AuthTypes } from './types';
+
+export function login(user: any): AuthTypes {
```

Comment:
export function login(users: any): AuthTypes { } --> users: any can be replaced by users: userType